src/Test.UnitTests.Sarif.Driver/DriverExtensionMethodsTests.cs

```diff
@@ -242,28 +242,53 @@ public void ValidatingMultipleFilesOutputOptions_ProducesExpectedResults()
         [Fact]
         public void ValidateAnalyzeOutputOptions_ProducesExpectedResults()
         {
-            AnalyzeOptionsBase analyzeOptionsBase = new TestAnalyzeOptions();
-            TestAnalysisContext context = new TestAnalysisContext();
+            var context = new TestAnalysisContext();
+            var analyzeOptionsBase = new TestAnalyzeOptions();
 
-            //  quiet false, output empty
+            // quiet false, output empty
             analyzeOptionsBase.Quiet = false;
             analyzeOptionsBase.OutputFilePath = null;
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet false, output non-empty
+            // quiet false, output non-empty
             analyzeOptionsBase.Quiet = false;
             analyzeOptionsBase.OutputFilePath = "doodle";
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet true, output empty
+            // quiet true, output empty
             analyzeOptionsBase.Quiet = true;
             analyzeOptionsBase.OutputFilePath = null;
             Assert.False(analyzeOptionsBase.ValidateOutputOptions(context));
 
-            //  quiet true, output non-empty
+            // quiet true, output non-empty
             analyzeOptionsBase.Quiet = true;
             analyzeOptionsBase.OutputFilePath = "doodle";
             Assert.True(analyzeOptionsBase.ValidateOutputOptions(context));
+
+            // postUri non-empty and outputFilePath empty = false
+            analyzeOptionsBase.PostUri = "doodle";
+            analyzeOptionsBase.OutputFilePath = null;
+            Assert.False(analyzeOptionsBase.ValidateOutputOptions(context));
+
+            // postUri valid non-empty and outputFilePath empty = false
+            analyzeOptionsBase.PostUri = "https://github.com/microsoft/sarif-sdk";
+            analyzeOptionsBase.OutputFilePath = null;
+            Assert.False(analyzeOptionsBase.ValidateOutputOptions(context));
+
+            // postUri non-empty and outputFilePath non-empty = true
```

Comment:
>// postUri non-empty and outputFilePath non-empty = true [](http://example.com/codeflow?start=12&length=56)

These comments aren't helpful. Why are you repeating the actual code in a comment?

Again, you are repeating a pattern. But you should always consider whether that pattern is worth repeat, don't just trust it because it exists. If there's a weakness in the pattern, then you are spreading it.